rasa/core/domain.py

```diff
@@ -103,6 +115,15 @@ def get_duplicate_exception_message(
                 )
         return message
 
+    def get_missing_templates(
+        action_names: List[Text], templates: Dict[Text, Any]
+    ) -> List[Text]:
+        """Return utterance names which have no specified template."""
+
+        utters = [act for act in action_names if act.startswith(action.UTTER_PREFIX)]
```

Comment:
lol looks like you were influenced by reading gabi's PR -- can we name this `utterances` please